src/main/java/core/basesyntax/strategy/impl/SupplyOperation.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.strategy.impl;

```

Comment:
The package declaration should not be followed by an empty line. Please remove the empty line after the package statement to adhere to the coding conventions.